meta.yaml

```diff
@@ -63,8 +63,9 @@ requirements:
     - ncurses=6.1=hfc679d8_1
     - gcc
     - r-base
-    - r-rcpp
+    - r-rcpp    
```

Comment:
There is no need to add extra whitespace here

Reply:
done